Build.ps1

```diff
@@ -1,24 +1,24 @@
-echo "build: Build started"
+Write-Output "build: Build started"
 
 Push-Location $PSScriptRoot
 
 if(Test-Path .\artifacts) {
-	echo "build: Cleaning .\artifacts"
+	Write-Output "build: Cleaning .\artifacts"
 	Remove-Item .\artifacts -Force -Recurse
 }
 
 & dotnet restore --no-cache
 
-$branch = @{ $true = $env:APPVEYOR_REPO_BRANCH; $false = $(git symbolic-ref --short -q HEAD) }[$env:APPVEYOR_REPO_BRANCH -ne $NULL];
-$revision = @{ $true = "{0:00000}" -f [convert]::ToInt32("0" + $env:APPVEYOR_BUILD_NUMBER, 10); $false = "local" }[$env:APPVEYOR_BUILD_NUMBER -ne $NULL];
+$branch = @{ $true = $env:APPVEYOR_REPO_BRANCH; $false = $(git symbolic-ref --short -q HEAD) }[$null -ne $env:APPVEYOR_REPO_BRANCH];
```

Comment:
What is the reason for changing the order of the arguments?

Reply:
it's a warning from the linter too, shipped within the `Powershell` extensions (msft powershell team)

i don't have the underlying reason for this one.
my personal guess would be that it applies the same principle as some unit test lib : expected/constants on the left operand.

but it's just a guest

Reply:
This reminds me https://en.wikipedia.org/wiki/Yoda_conditions

Reply:
i can totally revert it ;)
i just wanted to get rid of warning to be fair ^^

Reply:
It doesn’t matter to me.